net-mail/vpopmail/vpopmail-5.4.33-r4.ebuild

```diff
@@ -0,0 +1,300 @@
+		--enable-vpopuser=vpopmail \
+		--enable-vpopgroup=vpopmail \
+		--enable-many-domains \
+		--enable-file-locking \
+		--enable-file-sync \
+		--enable-md5-passwords \
+		--enable-logging \
+		--enable-auth-logging \
+		--enable-log-name=vpopmail \
+		--enable-qmail-ext \
+		--disable-tcpserver-file \
+		--disable-roaming-users \
+		$(use_enable ipalias ip-alias-domains) \
+		$(use_enable clearpasswd clear-passwd) \
+		$(use_enable maildrop) \
+		$(use_enable maildrop maildrop-prog /usr/bin/maildrop) \
+		$(use_enable spamassassin)
+}
+
+src_install() {
+	vpopmail_set_homedir
+
+	emake DESTDIR="${D}" install
+	keepdir "${VPOP_HOME}"/domains
+
+	# install helper script for maildir conversion
+	into "${VPOP_HOME}"
+	dobin "${FILESDIR}"/vpopmail-Maildir-dotmaildir-fix.sh
+	into /usr
+
+	einstalldocs
+	rm -rf "${D}/${VPOP_HOME}"/doc
+	dosym \
+		$(realpath --relative-to "${D}/${VPOP_HOME}"/ "${D}"/usr/share/doc/${PF}/) \
+		"${VPOP_HOME}"/doc
+
+	# create /etc/vpopmail.conf
+	if use mysql; then
+		dodir /etc
+		mv "${D}${VPOP_HOME}"/etc/vpopmail.mysql "${D}"/etc/vpopmail.conf
+		dosym \
+			$(realpath --relative-to "${D}/${VPOP_HOME}"/etc/ "${D}"/etc/vpopmail.conf) \
+			"${VPOP_HOME}"/etc/vpopmail.mysql
+
+		sed -e '12d' -i "${D}"/etc/vpopmail.conf
+		echo '# Read-only DB' >> "${D}"/etc/vpopmail.conf
+		echo 'localhost|0|vpopmail|secret|vpopmail' >> "${D}"/etc/vpopmail.conf
+		echo '# Write DB' >> "${D}"/etc/vpopmail.conf
+		echo 'localhost|0|vpopmail|secret|vpopmail' >> "${D}"/etc/vpopmail.conf
+
+		# lock down perms
+		fperms 640 /etc/vpopmail.conf
+		fowners root:vpopmail /etc/vpopmail.conf
+	fi
+
+	insinto "${VPOP_HOME}"/etc
+	doins vusagec.conf
+	dosym "${VPOP_HOME}"/etc/vusagec.conf /etc/vusagec.conf
+	sed -i 's/Disable = False;/Disable = True;/g' "${D}${VPOP_HOME}"/etc/vusagec.conf
+
+	einfo "Installing env.d entry"
+	dodir /etc/env.d
+	doenvd "${FILESDIR}"/99vpopmail
+
+	einfo "Locking down vpopmail permissions"
+	fowners -R root:0 "${VPOP_HOME}"/{bin,etc,include}
+	fowners root:vpopmail "${VPOP_HOME}"/bin/vchkpw
+	fperms 4711 "${VPOP_HOME}"/bin/vchkpw
+}
+
+pkg_postinst() {
+	if use mysql ; then
+		elog
+		elog "You have 'mysql' turned on in your USE"
+		elog "Vpopmail needs a VALID MySQL USER. Let's call it 'vpopmail'"
+		elog "You MUST add it and then specify its passwd in the /etc/vpopmail.conf file"
+		elog
+		elog "First log into mysql as your mysql root user and pass. Then:"
+		elog "> create database vpopmail;"
+		elog "> use mysql;"
+		elog "> grant select, insert, update, delete, create, drop on vpopmail.* to"
+		elog "  vpopmail@localhost identified by 'your password';"
+		elog "> flush privileges;"
+		elog
+		elog "If you have problems with vpopmail not accepting mail properly,"
+		elog "please ensure that /etc/vpopmail.conf is chmod 640 and"
+		elog "owned by root:vpopmail"
+		elog
+	fi
+
+	# do this for good measure
+	if [[ -e /etc/vpopmail.conf ]]; then
+		chmod 640 /etc/vpopmail.conf
```

Comment:
I think these commands need to die even when outside sandbox.

Reply:
same

Reply:
added ||die